h3/src/stream.rs

```diff
@@ -49,7 +174,7 @@ where
     stream: S,
     ty: Option<StreamType>,
     push_id: Option<u64>,
-    buf: BufList<S::Buf>,
+    buf: BufList<Bytes>,
```

Comment:
Also curious why this change.

Reply:
As argued in the last related comment, since `FrameStream` now stores `Bytes` instead of `S::Buf`, we'd need to map all the `S::Buf` into `Bytes` when constructing it via `FrameStream::with_bufs()`. To save up this iteration, best thing is to call `Buf::copy_to_bytes()` right after get our hands on each chunk.